### DISCUSSION TITLE
`dua a` will get the all the sizes of its content if the argv is one directory instead of the directory itself

### DISCUSSION THREAD

**Discussion by c02y** [Byron/dua-cli#237] [Ideas]
https://imgur.com/a/iO9m0n3

Would you please make `dua a` only get the size of the directory instead of the content, just like the argv is multiple directories?

**Comment by Byron** [comment]
It was working like proposed here until it was changed to what it is now. @piotrwach might know more about this.

As it's probably no good to flip-flop between subtleties, maybe there is a way to make the behaviour more consistent in a non-breaking way.
